### PR TITLE
Remove csw dependency

### PR DESCRIPTION
**Pull request by lforesta**
The EODC bindings do not connect to a CSW server to get a list of filepaths. This list is now to be provided as input to the Writer classes in this package.

### CODE REVIEW

**Comment by lforesta** [comment]
addresses #25

File: src/eodc_openeo_bindings/map_cubes_processes.py [hunk 1]
```diff
@@ -36,7 +25,8 @@ def map_load_collection(process):
         dict_item = map_filter_bbox(process)[0]
         dict_item_list.append(dict_item)
 
-    return dict_item_list, filepaths
+    # return dict_item_list, filepaths
```
Comment (sophieherrmann) [inline src/eodc_openeo_bindings/map_cubes_processes.py:28]: I think we can remove the commented line - it will remain in the git history anyway

**Review by sophieherrmann — COMMENTED** [review]
Looks good :)
Are there now any environment variables required? - Because we obviously don't need the `csw_url` any more

File: src/eodc_openeo_bindings/map_cubes_processes.py (hunk 1 above)
Comment (lforesta) [reply src/eodc_openeo_bindings/map_cubes_processes.py:28]: yes, forgot to clean that up

**Comment by lforesta** [comment]
> Are there now any environment variables required? - Because we obviously don't need the csw_url any more

Is this specified anywhere? I thought it was only in the csw function, which I have deleted

**Pull request merged**